app/views/posts/_editor.haml

```diff
@@ -1,3 +1,4 @@
+- character_galleries = item.character.galleries.includes(:icons).ordered if item.character && current_user.icon_picker_grouping? && item.character.galleries.count > 1
```

Comment:
I think you're going to double load icons here because the later line includes an 'order' which the `includes(:icons)` isn't using.